examples/rollup/rollup.bzl

```diff
@@ -11,17 +11,26 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+load("//:internal/collect_es6_sources.bzl", "collect_es6_sources")
+
 
 def _rollup(ctx):
   """This is not a full-featured Rollup bazel rule, just enough to test the
      other rules in this repo.
   """
-  args = ["--input", ctx.attr.entry_point]
+  inputs = collect_es6_sources(ctx)
+
+  entry_point = None
+  for i in inputs:
+    if ctx.attr.entry_point in i.path:
```

Comment:
what happens if we never set entry_point?

Reply:
There's now a cool error message